tests/integration/ceph_flex_test.go

```diff
@@ -173,8 +173,10 @@ func (s *CephFlexDriverSuite) statefulSetDataCleanup(poolName, storageClassName,
 	// Delete all PVCs
 	s.kh.DeletePvcWithLabel(defaultNamespace, statefulSetName)
 	// Delete storageclass and pool
-	s.testClient.PoolClient.DeletePool(s.testClient.BlockClient, s.clusterInfo, poolName)
-	s.testClient.BlockClient.DeleteStorageClass(storageClassName)
+	err := s.testClient.PoolClient.DeletePool(s.testClient.BlockClient, s.clusterInfo, poolName)
+	assertNoErrorUnlessNotFound(s.Suite, err)
+	err = s.testClient.BlockClient.DeleteStorageClass(storageClassName)
```

Comment:
How about ignoring the not found error inside the DeleteStorageClass() method instead? Then all the callers will benefit.